client/src/utils/Format/formatNumber.js

```diff
@@ -0,0 +1,18 @@
+const PLURAL_RULES = new Intl.PluralRules([], { type: 'ordinal' });
+const SUFFIXES = new Map([
+  ['one', 'st'],
+  ['two', 'nd'],
+  ['few', 'rd'],
```

Comment:
Given the first two maps should the third one not be three instead of few?
```suggestion
  ['three', 'rd'],
```

Reply:
I thought the same thing, @keenan218. I changed it at first myself but that's apparently not how `Intl.PluralRules` works. "few" = "three" for some reason. No idea why. 🤷‍♂️

Reply:
Ah okay, if it doesn't like the use of 'three' we shall just stick with 'few', as long as it works the same i guess it doesn't matter too much...